scw/client.go

```diff
@@ -340,6 +351,182 @@ func (c *Client) doListAll(req *ScalewayRequest, res interface{}) (err error) {
 	return errors.New("%T does not support pagination", res)
 }
 
+// doListLocalities collects all localities using mutliple list requests and aggregate all results on a lister response
+// results is sorted by locality
+func (c *Client) doListLocalities(req *ScalewayRequest, res lister, localities []string) (err error) {
```

Comment:
Can you add an example to the Getting Stated Readme

Reply:
Please check the examples here https://github.com/scaleway/scaleway-sdk-go/blob/v1.0.0-beta.9/example_test.go#L160